### PR TITLE
Email Confirmation

### PR DESCRIPTION
**Pull request by lequytra**

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -28,4 +28,10 @@ def formatted_phone_number
       "(#{area_code}) #{prefix}-#{suffix}"
     end
   end
+
+  protected
+   # Comment this out if we want to enforce email confirmation
+  def confirmation_required?
+    false
+  end
```
Comment (AlexanderOtavka) [inline app/models/user.rb]: What happens if someone's time to confirm runs out, but this is set false?

File: config/initializers/devise.rb [hunk 2]
```diff
@@ -18,7 +18,7 @@
   # Configure the e-mail address which will be shown in Devise::Mailer,
   # note that it will be overwritten if you use your own mailer class
   # with default "from" parameter.
-  config.mailer_sender = "change-password@#{ENV['MAILGUN_DOMAIN']}"
+  config.mailer_sender = "account-updates@#{ENV['MAILGUN_DOMAIN']}"
```
Comment (AlexanderOtavka) [inline config/initializers/devise.rb]: Good!

File: config/initializers/devise.rb [hunk 3]
```diff
@@ -131,15 +131,15 @@
   # without confirming their account.
   # Default is 0.days, meaning the user cannot access the website without
   # confirming their account.
-  # config.allow_unconfirmed_access_for = 2.days
+   config.allow_unconfirmed_access_for = 2.days
```
Comment (AlexanderOtavka) [inline config/initializers/devise.rb]: Is this overridden by `User#confirmation_required?`?

File: config/routes.rb [hunk 4]
```diff
@@ -1,7 +1,7 @@
 Rails.application.routes.draw do
   root to: "welcome#index"
 
-  devise_for :users
+  devise_for :user, path: '/devise'
```
Comment (AlexanderOtavka) [inline config/routes.rb]: Is this necessary?  Do emails still work without this change?

**Comment by AlexanderOtavka** [comment]
Also, I did a rebase and a force push to get the diff to be nice.  Be sure to pull before making changes.  Hopefully this doesn't screw up your workflow.

File: config/initializers/devise.rb (hunk 3 above)
Comment (lequytra) [reply config/initializers/devise.rb]: Yes

**Comment by AlexanderOtavka** [comment]
Because this branch is so far behind, merging in the migration will be a bit tricky.  Don't bother manually updating schema.rb, it is auto-generated.  Instead:

1. Go to master and pull down the latest changes
2. Do `rails db:schema:load` to put your schema in line with the production schema
3. Go to this confirmation branch
1. Merge from master and fix the conflicts in routes.rb
2. Modify the file name of your migration so that it becomes the most recent one, but still has a date in the past so new migrations will go on top of it.  Probably just copy the number of the previous one and add 1.
6. Run `rails db:migrate` to apply your migration and overwrite the schema.rb file
7. Add and commit the newly generated schema.rb to resolve the conflict

**Pull request closed without merging**